components/core/src/fs.rs

```diff
@@ -199,7 +241,13 @@ pub fn find_command(command: &str) -> Option<PathBuf> {
             for path in env::split_paths(&paths) {
                 let candidate = PathBuf::from(&path).join(command);
                 if candidate.is_file() {
+                    println!("Found {:?}", candidate);
```

Comment:
This looks like it's either debug output or something left over from iterating. If it's debug output let's use `debug!()` instead of `println!()` and if it's just left over from iterating remove it